spec/models/tutors_spec.rb

```diff
@@ -22,12 +22,31 @@
   describe 'validations' do
     describe 'name' do
       it 'validates presence' do
-        should validate_presence_of(:name)
+        should validate_presence_of(:first_name)
+        should validate_presence_of(:last_name)
       end
+    end
 
-      it 'validates length' do
-        should validate_length_of(:name)
-          .is_at_most(50)
+    describe 'phone numbers' do
+      it 'validates format' do
+        should allow_value('(555) 555-5555').for(:phone)
+        should allow_value('555-555-5555').for(:phone)
+        should_not allow_value('43-4343-4444').for(:phone)
+        should_not allow_value('434-3433-4444').for(:phone)
+        should_not allow_value('434-434-444').for(:phone)
+        should_not allow_value('4333-433-4444').for(:phone)
+        should_not allow_value('(232) 343-99439').for(:phone)
+        should_not allow_value('(abc) 123-defg').for(:phone)
+      end
+    end
+
+    describe 'zip' do
+      it 'validates format' do
```

Comment:
That's probably a later ticket. I couldn't find a straight forward answer from a few google searches about how to cap it. Also would need some input from Roberta for how big a radius is allowed